src/signer_storage/signer_list.cairo

```diff
@@ -189,7 +177,7 @@ mod signer_list_component {
         ) -> bool {
             let mut is_before: bool = false;
             let mut current_signer = first_signer;
-            loop {
+            while true {
```

Comment:
I would let `loop` instead of `while true` mainly to tell that breaking conditions are inside the loop body ("old fashion Cairo loop").